packages/monaco/src/browser/monaco-editor-zone-widget.ts

```diff
@@ -41,9 +41,12 @@ export class MonacoEditorZoneWidget implements Disposable {
         this.toHide
     );
 
+    readonly editorLineHeight: number;
```

Comment:
i don't think it should be like that, we already capture height in zone object and options are always available from preferences.